perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -115,7 +120,7 @@ def _export_to_gsheet(folders, metrics, command, worksheet) -> None:
   return
 
 
-def _parse_results(folders, results_list, message, num_samples) -> dict:
+def _parse_results(folders, result_list, message, num_samples) -> dict:
```

Comment:
Now, this method will take list containing run_interval for every execution for the specified folder. Right? 
Let's rename it to run_interval_list.

Reply:
It's a dictionary that has a list of start and end time pairs for each sample run. How about naming it time_intervals_dict?